examples/src/utils/renderer-debug-widget.ts

```diff
@@ -256,12 +256,14 @@ function updateSceneInfo(scene: THREE.Object3D, sceneInfo: SceneInfo) {
     materials.forEach(m => materialIds.add(m.id));
 
     if (x.name.startsWith('Primitives')) {
+      const indexAttributes = geometry.index!;
```

Comment:
Any reason why assigning a new const instead of calling geometry.index!.count?

Reply:
Mja, Law of Demeter and readability. I think its more explicitly visible that we perform an explicit not-null cast when the ! is the last character.